sktime/transformations/panel/tests/test_derivative.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+"""Test derivative transformer."""
+import numpy as np
+
+from sktime.distances.tests._utils import create_test_distance_numpy
+from sktime.transformations.panel.derivative import DerivativeTransformer
+
+
+def test_derivative():
```

Comment:
is this not already being tested by the generic tests?

Reply:
I didn't realise there was generic testing going on in this module Ill remove this then and leave it to that!